### PR TITLE
add skipNotFound prop

### PR DESCRIPTION
**Pull request by FullMoon04**
Attempt to implement #30

### CODE REVIEW

File: src/TourPortal.js [hunk 1]
```diff
@@ -97,6 +99,9 @@ class TourPortal extends Component {
       inDOM: false,
       observer: null,
     }
+
+    const { skipNotFound, steps } = this.props
+    this.steps = skipNotFound ? this.filterSteps(steps) : steps
```
Comment (elrumordelaluz) [inline src/TourPortal.js]: I'm not sure if is the right place to check/filter if DOM nodes are present.

**Comment by elrumordelaluz** [comment]
hey @FullMoon04 thansks again to take an _issue_ and implement a solucion, apreciate a lot!

Everything seems ok for me but I have a doubt about the place where check if `node` exists, since you are doing it in the `constructor` (the first method in the _Component_ lifecycle order) and maybe node it isn't rendered yet.

As @hip1310 said [here](https://github.com/elrumordelaluz/reactour/issues/30#issue-301588948)
> We have some divs which are rendered only sometimes.

I suspect that no every _step_ is ready/rendered (present in the DOM to check `document.querySelector()`) when the _Tour_ Compoent mount.

Let me know if make sense for you, and thanks again!

File: src/TourPortal.js (hunk 1 above)
Comment (FullMoon04) [reply src/TourPortal.js]: I am sorry but I do not understand well either.

**Comment by FullMoon04** [comment]
Certainly I think you are right.
> I suspect that no every step is ready/rendered (present in the DOM to check document.querySelector()) when the Tour Compoent mount.

I have little experience in React. I'll let you know if I learn anything more.
Thank you for your review.

**Comment by elrumordelaluz** [comment]
No problem at all. PR super welcome!

One idea could be to check if node exists just before change into it. There is a [similar approach](https://github.com/elrumordelaluz/reactour/blob/master/src/TourPortal.js#L172) to check if node exists and calculate the _helperp_ position (in case doesn't exists shows in the middle of the screen).

**Pull request closed without merging**